project_types/base/project.py

```diff
@@ -342,18 +342,18 @@ def handle_project_update_on_firebase(self, project_ref: FbReference, fb_project
                     image=self.project.image.file.url if self.project.image else firebase_models.UNDEFINED,
                     isFeatured=self.project.is_featured,
                     lookFor=self.project.look_for,
-                    name=self.project.name,
-                    # FIXME(tnagorra): Fill these when implemented
-                    projectNumber=1,
-                    projectRegion="",
-                    projectTopic="",
+                    name=self.project.generated_name,
+                    projectNumber=self.project.project_number,
+                    projectRegion=self.project.region,
+                    projectTopic=self.project.topic,
+                    # NOTE: projectTopicKey is not obsolete
```

Comment:
```suggestion
                    # NOTE: projectTopicKey is obsolete
```

My bad. projectTopicKey is obsolete